IHP/Controller/Param.hs

```diff
@@ -300,6 +300,16 @@ instance ParamReader Double where
                     Right integer -> Right (fromIntegral integer)
     readParameterJSON _ = Left "ParamReader Double: Expected Double"
 
+instance ParamReader Scientific.Scientific where
```

Comment:
Tests for ParamReader instances can be added here https://github.com/digitallyinduced/ihp/blob/master/Test/Controller/ParamSpec.hs :) Could you add a case there? :)